client/src/elements/Background.js

```diff
@@ -1,21 +1,43 @@
-import React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles((theme) => ({
   gradient: {
-    height: '100vh',
-    overflowY: 'auto',
+    height: "100vh",
+    overflowY: "auto",
     background: theme.palette.background.gradient,
-    display: 'flex'
+    display: "flex",
   },
   solid: {
-    height: '100vh',
+    height: "100vh",
     background: theme.palette.background.solid,
-    display: 'flex'
-  }
+    display: "flex",
+  },
+  inApp: {
+    height: "100vh",
+    overflowY: "auto",
+    background: "#ecf0fa",
+    display: "flex",
+  },
+  inAppMenu: {
+    height: "100vh",
+    overflowY: "auto",
+    background: "#ffffff",
+    display: "flex",
+  },
 }));
 
 export default function Background(props) {
   const classes = useStyles();
-  return <div className={props.gradient ? classes.gradient : classes.solid}>{props.children}</div>;
+
+  // Need to implement switch for in app themes
+  const getBackroundType = (type) => {
+    const backgrounds = {};
+  };
```

Comment:
I don't think you are using this at the moment right? Don't know exactly it's purpose at the moment

Reply:
Will remove from code, as yes this is not used.

Reply:
Removed